api/config/config_test.json

```diff
@@ -49,7 +49,7 @@
     "extractionPetServiceAccount": "pet-118217329794842274136@aou-wgs-cohort-extraction.iam.gserviceaccount.com",
     "extractionMethodConfigurationNamespace": "aouwgscohortextraction",
     "extractionMethodConfigurationName": "HelloWorld",
```

Comment:
Probably not in this PR: is there a plan to fix this at some point? "HelloWorld" is a bit of an eye sore..